<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1527</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Найти похожие лица</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="targetFaceFileChooseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetFaceImageLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите изображение, где находится целевое лицо</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="directoryChooseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать папку</string>
    </property>
   </widget>
   <widget class="QLabel" name="directoryPathLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Папка не выбрана</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startAnalyzeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать анализ</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="analyzeProgressBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="analyzeProgressLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0 Обработано</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="foundFaceImgsScrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>1511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1509</width>
       <height>449</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1527</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
